### DISCUSSION TITLE
KeyError: 'default_vocoder' when run server with voice_conversion_models/multilingual/vctk/freevc24

### DISCUSSION THREAD

**Discussion by nghidinhit** [coqui-ai/TTS#2886] [General Q&A]
I met this error when run server with voice_conversion_models/multilingual/vctk/freevc24 model.
Please help.

**Comment by nghidinhit** [comment]
When I run as the following:

docker run --rm -it -p 5002:5002 --entrypoint /bin/bash ghcr.io/coqui-ai/tts-cpu
python3 TTS/server/server.py --model_name
voice_conversion_models/multilingual/vctk/freevc24

I met the error:
Traceback (most recent call last):
  File "/root/TTS/server/server.py", line 88, in <module>
    args.vocoder_name = model_item["default_vocoder"] if
args.vocoder_name is None else args.vocoder_name
KeyError: 'default_vocoder'

**Reply by rohitkrishna094** [reply]
How did you fix this?

**Reply by krushnapatel-qb** [reply]
did you find any solution regarding to this problem???

**Reply by eginhard** [reply]
I think the server so far only supports (some) TTS models, not VC. PRs welcome in our fork: https://github.com/idiap/coqui-ai-TTS
